src/config/config.go

```diff
@@ -42,3 +44,11 @@ func Parse(config []byte) (cfg Config, err error) {
 	}
 	return
 }
+
+func (cfg Config) WriteFile(path string, mode os.FileMode) (err error) {
```

Comment:
I don't follow. A view the config cache as an opaque blob; the same blob that was downloaded. Given that, it should be fine to pass the blob to config.Parse().